### ISSUE TITLE
Go to functionality for regions

### ISSUE THREAD

**Issue by Nizari** [maptz/maptz.vscode.extensions.customfolding#61]
Hello, love the tool./ Was wondering if it's also possible to go to the start of a region? Like with go to symbol.

**Comment by maptz** [comment]
Yes. This should be easily doable. At which point would you suggest though, [1] or [2] below:

```
[1]//#region Some Region
[2]public class SomeClass
{
}
//#endregion
```

**Comment by Nizari** [comment]
I would say point 1. The start of the region. Great! Would be great feature I think. Thank you

**Comment by maptz** [comment]
This is a doable enhancement. I'll look at this next

**Comment by HerrJensen81** [comment]
I don't know if it fits to this topic.
I would like to jump to regions by using a "region naviagtion"
My idea is that the extension adds an Icon to the "Activitay bar".
Then inside the "Side Bar" you can see a tree view of the regions.
A click on the region will jump to the region in the code.

**Comment by maptz** [comment]
Have you tried enabling the sidebar?
